CRM/Queue/Queue/SqlTrait.php

```diff
@@ -73,13 +73,17 @@ public function numberOfItems() {
    * @param array $options
    *   Queue-dependent options; for example, if this is a
    *   priority-queue, then $options might specify the item's priority.
+   *   Ex: ['release_time' => strtotime('+3 hours')]
    */
   public function createItem($data, $options = []) {
     $dao = new CRM_Queue_DAO_QueueItem();
     $dao->queue_name = $this->getName();
     $dao->submit_time = CRM_Utils_Time::getTime('YmdHis');
     $dao->data = serialize($data);
     $dao->weight = CRM_Utils_Array::value('weight', $options, 0);
+    if (isset($options['release_time'])) {
+      $dao->release_time = date('Y-m-d H:i:s', $options['release_time']);
```

Comment:
Fun fact (*and pre-existing issue*): `civicrm_queue_item.release_time` uses `DATETIME`. This is ambiguous - there is no clear definition of the *storage-time-zone*.

Of course, AFAIK no one has been complaining about that. I suppose a lot of queue-stuff is done by background workers which (for a given deployment) will have consistent/mutually-agreeable TZ settings. (*Or maybe it's just hard to notice - how many people will recognize when the lock-release of a failed background task is wrong by a couple hours?*)

The calculation here should be *as good as* any existing calculations on this column (as performed in [this recipe](https://docs.civicrm.org/dev/en/latest/framework/queues/#2-create-itemstasks-on-the-queue) or in core) -- ie it's OK for simple deployments with relatively consistent timezones, but it's highly suspect for multi-time-zone usage.

All of that really should be a separate issue/bug. But it's a bit relevant in how it influences the interface of `$options['release_time']`. In this patch:

* `$options['release_time']` is __not__ `string` (`Y-m-d H:i:s`) because that is ambiguous. (It is unclear whether the timezone of `Y-m-d H:i:s` should be the current-user-TZ, the queue-worker-TZ, the server-TZ, or GMT.)
* `$options['release_time']` is `int` (seconds since epoch) because that is unambiguous.